hack/bin/selfsigned.sh

```diff
@@ -19,23 +19,54 @@ echo '{
         "algo": "rsa",
         "size": 2048
     }
-}' > "$CSR"
+}' >"$CSR"
 
 # generate CA key and cert
 cfssl gencert -initca "$CSR" | cfssljson -bare "$OUTPUTNAME_CA"
+# TODO: only generate CA once!
```

Comment:
all changes to this file should be reverted, since there is a selfsigned-kubernetes file.

Reply:
Done in 2f72d3988668a4ba4a50fb9634fffb4bf7d9a8d2